### PR TITLE
[VEN-2527]: Chaos Lab recommendation for the week of April 4, 2024.

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves VEN-2527

### CODE REVIEW

File: vips/vip-285/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        params: [VUSDT_GameFi_IR],
+      },
+      {
+        target: VUSDT_DeFi,
+        signature: "setInterestRateModel(address)",
+        params: [VUSDT_DeFi_IR],
+      },
+    ],
+    meta,
+    ProposalType.REGULAR,
```
Comment (chechu) [inline vips/vip-285/bscmainnet.ts:87]: Let's do it fast-track

File: multisig/simulations/ethereum/vip-017/index.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+const vCURUSD_CORE = "0x672208C10aaAA2F9A6719F449C4C8227bc0BC202";
+const vCURUSD_CURVE = "0x2d499800239C4CD3012473Cb1EAE33562F0A6933";
+
+const NEW_IR = "0xd7fbFD2A36b8b388E6d04C7a05956Df91862E146";
+const OLD_IR = "0x1C243a1aCe202424fa79F71de36225DF93B9e5C5";
+forking(19612715, () => {
+  let vusdc: Contract;
+  let vusdt: Contract;
+  let vcurusdCore: Contract;
+  let vcurvusdCurve: Contract;
```
Comment (chechu) [inline multisig/simulations/ethereum/vip-017/index.ts:22]:
```suggestion
  let vcurusdCurve: Contract;
```

**Review by chechu — COMMENTED** [review]
lgtm

File: vips/vip-285/bscmainnet.ts (hunk 1 above)
Comment (GitGuru7) [reply vips/vip-285/bscmainnet.ts]: 0c10417f73da1cdbe2c2e0f4beee2e3b3f64dbfb

File: multisig/proposals/ethereum/vip-017/index.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { makeProposal } from "../../../../src/utils";
+
+const vUSDC_CORE = "0x17C07e0c232f2f80DfDbd7a95b942D893A4C5ACb";
+const vUSDT_CORE = "0x8C3e3821259B82fFb32B2450A95d2dcbf161C24E";
+const vCURUSD_CORE = "0x672208C10aaAA2F9A6719F449C4C8227bc0BC202";
+const vCURUSD_CURVE = "0x2d499800239C4CD3012473Cb1EAE33562F0A6933";
+
+const NEW_IR = "0xd7fbFD2A36b8b388E6d04C7a05956Df91862E146";
+const vip016 = () => {
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply multisig/proposals/ethereum/vip-017/index.ts]: 0c10417f73da1cdbe2c2e0f4beee2e3b3f64dbfb

File: multisig/proposals/opbnbmainnet/vip-006/index.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { makeProposal } from "../../../../src/utils";
+
+const vFDUSD_CORE = "0x13B492B8A03d072Bab5C54AC91Dba5b830a50917";
+const vUSDT_CORE = "0xb7a01Ba126830692238521a1aA7E7A7509410b8e";
+
+const NEW_IR = "0xaf6862b20280818FA24fA6D17097517608Fe65d4";
+const vip006 = () => {
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply multisig/proposals/opbnbmainnet/vip-006/index.ts]: 0c10417f73da1cdbe2c2e0f4beee2e3b3f64dbfb

File: multisig/simulations/ethereum/vip-017/index.ts (hunk 2 above)
Comment (GitGuru7) [reply multisig/simulations/ethereum/vip-017/index.ts]: 0c10417f73da1cdbe2c2e0f4beee2e3b3f64dbfb

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/288?chainId=56

**Pull request merged**